_posts/2019-02-07-checking-the-status-of-your-api-submitted-deal.md

```diff
@@ -0,0 +1,157 @@
+in your request headers such that the request header key is `api_key` and the
+request header value for that key is your **API key**. If you omit the api_key or
+provide an incorrect one, the HTTP status code of the response will be 401
+Unauthorized.
+
+#### HTTPS
+
+Since your API requests will contain sensitive personal information about our
+customers, you must use SSL (HTTPS). To discourage insecure API requests,
+the response code will be 403 Forbidden for all requests made over HTTP.
+
+#### Rate Limiting
+
+We impose a rate limit per user of 1 request per second. If you exceed this rate
+of requests, the response code will be 429 Too Many Requests. If you need to
+exceed this limit for some reason, please let us know!
+
+## The Deal Status Endpoint
+
+You will be able to check your deal status with a GET request to the `/v1/deal_status` 
+endpoint with your **lead ID**.
+For example, the GET url might be
+
+`https://api-staging.forwardfinancing.com/v1/deal_status/<LEAD_ID>`
+
+### Request
+
+The only thing you'll need in the request header is your **API key**!
+
+```
+api_key: YOUR_API_KEY_HERE
+```
+
+### Response
+The possible response body fields are 
+- stage
+- maxApproval
+- maxPayments
+- offerLink
+- declineDrivers
+- declineNotes
+- missingInfo
+
+Your response will contain different fields depending on the status of your deal.
+All responses will contain the **stage** of your deal.
+
+|**Deal Status**|stage  | maxApproval   | maxPayments  | offerLink  | declineDrivers  | declineNotes  | missingInfo |
+|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
+|   |   |   |   |   |   |   |   |
+| We're currently processing your deal  | x |  |  |   |   |   |   |
+|   |   |   |   |   |   |   |   |
+| Your deal has been approved  | x | x | x | x |   |   |   |
+|   |   |   |   |   |   |   |   |
+| Your deal has been declined  | x |   |   |   | x | x |   |
+|   |   |   |   |   |   |   |   |
+| We need more information  | x |   |   |   |   |   | x |
+
+## Glossary
+**stage** - the current state of your submitted deal in our process 
+
+**maxApproval** - the maximum advance amount approved for this merchant
+
+**maxPayments** - the maximum number of payments approved for this merchant (term in days), (we arrived that by multiplying months by 20) 
+
+**offerLink** - link to our offer calculator where you can select terms and request contracts
+
+**declineDrivers** - reasons why this deal was declined chosen from a set list (see below for list)
+
+**declineNotes** - additional explanation for why this deal was declined 
+
+**missingInfo** - additional information that is needed for a decision to  be made on this deal
+
+
+## Sample Responses 
+These are some possible responses you may receive from our API
+```
+{
+  "stage": "Approval Sent",
+  "maxApproval": "10000",
+  "maxPayments": "10",
+  "offerLink": "offer.com"
+}
+```
+
+```
+{
+  "stage": "Declined",
+  "declineDrivers": [Default history, Excessive liens or judgements],
```

Comment:
`declineDrivers` is an array of strings? Make sure all these JSON examples are valid.